### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by igor-stasiv**

### CODE REVIEW

**Review by Ihor-Karpyn — APPROVED** [review]
In this case use switch is better.  Everything else looks good

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Agreed. Let's use `switch/case` structure for this task.

File: src/transformState.js [hunk 1]
```diff
@@ -62,7 +62,28 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const { operation, properties } of transforms) {
```
Comment (VinnieJ-2k20) [inline src/transformState.js:65]: 👍

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**